### PR TITLE
updated the readme and script to align with the latest changes

### PR DESCRIPTION
**Pull request by jeroendekorte**
Guys, 

I've updated the readme to align with the latest changes. Please merge.

Rergard,
Jeroen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,44 +9,65 @@ Cloudify's CloudStack plugin provides the ability to create [CloudStack](http://
...
+cd ~
+virtualenv cfy-demo
+cd ~/cfy-demo
+source bin/activate
+pip install https://github.com/cloudify-cosmo/cloudify-dsl-parser/archive/master.zip
+pip install https://github.com/cloudify-cosmo/cloudify-rest-client/archive/master.zip
+pip install https://github.com/cloudify-cosmo/cloudify-plugins-common/archive/master.zip
+pip install https://github.com/cloudify-cosmo/cloudify-script-plugin/archive/master.zip
+pip install https://github.com/cloudify-cosmo/cloudify-cli/archive/master.zip
+git clone https://github.com/schubergphilis/cloudify-manager-blueprints.git
```
Comment (iliapolo) [inline README.md]: please only use cloudify-cosmo links...
i know it was always like this, but still, now seems like a good chance to fix it.

**Comment by jeroendekorte** [comment]
Sharp ;) Updated it.. thanks Eli

**Comment by idanmo** [comment]
Hi,
There are merge conflicts, please resolve them.

**Comment by jeroendekorte** [comment]
After 6 months I guess that there are some merge conflicts. I closed the PR.

**Pull request closed without merging**